en/config_rover/mecanum.md

```diff
@@ -0,0 +1,298 @@
+   :::
+
+The rover is now ready to drive in [Position mode](../flight_modes_rover/mecanum.md#position-mode).
+
+## Auto Modes
+
+::: warning
+For this mode to work properly [Acro mode](#acro-mode), [Stabilized mode](#stabilized-mode) and [Position mode](#position-mode) must already be configured!
+:::
+
+<a id="pure_pursuit_controller"></a>
+In [auto modes](../flight_modes_rover/mecanum.md#auto-modes) the autopilot takes over navigation tasks using the following control architecture:
+
+![Pure Pursuit Controller](../../assets/airframes/rover/rover_mecanum/auto_control_structure_mecanum.png)
+
+The mecanum module fully leverages the omnidirectionality of this type of rover by maintaining the initial heading of the rover during the entire mission (see [Pure Pursuit Guidance Logic](#pure-pursuit-guidance-logic)).
+
+The required parameters are separated into the following sections:
+
+### Velocity
+
+These parameters are used to calculate the velocity setpoint in auto modes:
+
+1. [RM_MISS_SPD_DEF](#RM_MISS_SPD_DEF): Sets the default speed ($m/s$) for the rover during the mission. For mecanum rovers the speed is defined in the direction of travel (magnitude of the velocity vector consisting of the forward and lateral speed).
+2. [RM_MAX_ACCEL](#RM_MAX_ACCEL) ($m/s^2$) and [RM_MAX_JERK](#RM_MAX_JERK) ($m/s^3$) are used to calculate a velocity trajectory such that the rover comes to a smooth stop as it reaches a waypoint.
+
+   ::: tip
+   Plan a mission for the rover to drive a square and observe how it slows down when approaching a waypoint.
+   If the rover decelerates too quickly decrease the [RM_MAX_ACCEL](#RM_MAX_ACCEL) parameter, if it starts slowing down too early increase the parameter.
+   If you observe a jerking motion as the rover slows down, decrease the [RM_MAX_JERK](#RM_MAX_JERK) parameter otherwise increase it as much as possible as it can interfere with the tuning of [RM_MAX_ACCEL](#RM_MAX_ACCEL).
+   These two parameters have to be tuned as a pair, repeat until you are satisfied with the behaviour.
+   :::
+
+4. [RM_MISS_VEL_GAIN](#RM_MISS_VEL_GAIN): The rover slows down when approaching a waypoint based on the angle between the line segment from the previous to current waypoint and current to next waypoint. How much the rover slows down can be tuned with this parameter:
+   
+   $$
+    v_{transition} = v_{max} \cdot (1 - \theta_{normalized} * k)
+   $$
+with $v_{transition}:$ Transition velocity, $v_{max}:$  Maximum speed ([RM_MISS_SPD_DEF](#RM_MISS_SPD_DEF)), $\theta_{normalized}:$ Angle between the linesegment of prev-curr and curr-next waypoints normalized from $[0\degree, 180\degree]$ to $[0, 1]$ and $k:$ Tuning parameter ([RM_MISS_VEL_GAIN](#RM_MISS_VEL_GAIN)).
+
```

Comment:
This renders hard to read. Perhaps put the values in a bullet list?

Reply:
Updated in https://github.com/PX4/PX4-user_guide/pull/3411/commits/c5045f58c7e90e9b80aa7d2641ca1c564476e6bc